### PR TITLE
[DE] Add `Marathon`

### PR DESCRIPTION
**Pull request by FreezyLemon**
I hope it's fine to leave that TODO comment there.

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/Marathon/de.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Marathon
+
+Der Begriff **Marathon** wird verwendet, um [Beatmaps](/wiki/Beatmap) mit einer Länge von mehr als fünf Minuten zu beschreiben. Diese Beatmaps sind oft [Kollaborationen](/wiki/Beatmap/Beatmap_collaborations), da das Mapping von längeren Songs für einen einzelnen Mapper anstrengend sein kann.
```
Comment (Deimosu) [inline wiki/Beatmap/Marathon/de.md:7]:
```suggestion
Der Begriff **Marathon** wird verwendet, um [Beatmaps](/wiki/Beatmap) mit einer Länge von mindestens fünf Minuten zu beschreiben. Diese Beatmaps sind oft [Kollaborationen](/wiki/Beatmap/Beatmap_collaborations), da das Mapping von längeren Songs für einen einzelnen Mapper anstrengend sein kann.
```

File: wiki/Beatmap/Marathon/de.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Geschichte
+
+Als Marathon-Beatmaps zuerst eingeführt wurden, mussten sie mindestens *sechs* Minuten lang sein und durften nur eine einzelne Schwierigkeitsstufe enthalten (mit der Ausnahme, dass jede Beatmap noch eine zusätzliche [osu!taiko](/wiki/Game_mode/osu!taiko)-Schwierigkeitsstufe beinhalten konnte).
+
+Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Submission-System zu umgehen und nicht in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet zu werden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Später wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
```
Comment (Deimosu) [inline wiki/Beatmap/Marathon/de.md:13]:
```suggestion
Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Submission-System zu umgehen und zu verhindern, dass die Beatmaps in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet wurden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Später wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
```

File: wiki/Beatmap/Marathon/de.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Später wurde eingeführt, dass auch Beatmaps mit fünf Minuten an [Drain-Zeit](/wiki/Beatmap/Drain_time) als Marathon-Beatmaps zählen und dass [benutzerdefinierte Namen für Schwierigkeitsgrade](/wiki/Ranking_criteria/Difficulty_naming) verwendet werden können. Dieses System blieb bis 2019 gleich und wurde dann von den weniger strengen Regeln zu Schwierigkeits-Spreads ersetzt. <!-- TODO: Update the translation for "Schwierigkeits-Spreads" when the article for it exists -->
```
Comment (Deimosu) [inline wiki/Beatmap/Marathon/de.md:15]:
```suggestion
Später wurde eingeführt, dass auch Beatmaps mit fünf Minuten an [Drain-Zeit](/wiki/Beatmap/Drain_time) als Marathon-Beatmaps zählen und dass [benutzerdefinierte Namen für Schwierigkeitsstufen](/wiki/Ranking_criteria/Difficulty_naming) verwendet werden können. Dieses System blieb bis 2019 gleich und wurde dann von den weniger strengen Regeln zu Schwierigkeits-Spreads ersetzt. <!-- TODO: Update the translation for "Schwierigkeits-Spreads" when the article for it exists -->
```

**Review by Deimosu — CHANGES_REQUESTED** [review]

File: wiki/Beatmap/Marathon/de.md (hunk 2 above)
Comment (FreezyLemon) [reply wiki/Beatmap/Marathon/de.md:13]: Is there a problem with `zu werden` here? I'd prefer something like this:

```suggestion
Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Submission-System zu umgehen und um zu verhindern, in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet zu werden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Später wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
```

File: wiki/Beatmap/Marathon/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Marathon/de.md:7]: To not have `mindestens` twice in the article, you could also write:
```suggestion
Der Begriff **Marathon** wird verwendet, um [Beatmaps](/wiki/Beatmap) zu beschreiben, die länger als 5 Minuten sind. Diese Beatmaps sind oft [Kollaborationen](/wiki/Beatmap/Beatmap_collaborations), da das Mapping von längeren Songs für einen einzelnen Mapper anstrengend sein kann.
```

File: wiki/Beatmap/Marathon/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Marathon/de.md:13]: Both of the suggestions above are fine to me.

There's something else I would like to mention.

```suggestion
Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Einreichungssystem zu umgehen und um zu verhindern, in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet zu werden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Schlussendlich wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
```

Firstly, I translated BSS in [Beatmapping/Beatmap submission](https://osu.ppy.sh/wiki/de/Beatmapping/Beatmap_submission) with `Beatmap-Einreichungssystem`.

Secondly, `Später` could be changed to `Schlussendlich` to have more word variety.

File: wiki/Beatmap/Marathon/de.md (hunk 3 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Marathon/de.md:15]: My suggestion is to use `festgelegt` instead of `eingeführt`.

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]

**Comment by FreezyLemon** [comment]
After some discussion on Discord, it seems like the accepted translation for `Difficulty spread` is going to be `Schwierigkeitsverteilung`. So I inserted it and removed the TODO comment.

Also applied review suggestions

**Review by Deimosu — APPROVED** [review]

**Review by The-Last-Cookie — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**